autoprotocol/compound.py

```diff
@@ -0,0 +1,12 @@
+class Compound:
+    """
+    Represents a single Compound
+
+    Parameters
+    ----------
+    inchi : Standard International Chemical Identifier
+
+    """
+
+    def __init__(self, inchi):
+        self.inchi = inchi
```

Comment:
Could we add a simple validation here? There are some simple regex validations available online.